aws_config.go

```diff
@@ -79,7 +78,19 @@ func GetAwsConfig(ctx context.Context, c *Config) (aws.Config, error) {
 		config.WithEC2IMDSClientEnableState(imdsEnableState),
 		config.WithHTTPClient(httpClient),
 		config.WithAPIOptions(apiOptions),
+		// FIXME: This should only be set for retrieving Creds
+		config.WithRetryer(func() aws.Retryer {
+			return aws.NopRetryer{}
+		}),
```

Comment:
Does this mean that no retries are attempted for any service?

Reply:
For now, though it will be only when retrieving credentials. I'm currently doing some refactoring to resolve that